File: c#/tema_2/ConcertPro/Dao/Repository/SpectacleRepository/ISpectacleRepository.cs
using System;
using System.Collections.Generic;
using ConcertPro.Dao.Model;

namespace ConcertPro.Dao.Repository.SpectacleRepository
{
    public interface ISpectacleRepository : IRepository<Spectacle, long>
    {
        IEnumerable<Spectacle> GetAllByDate(DateTime date);
    }
}

File: java/tema_2/src/main/java/ro/mpp2024/concertpro/dao/repository/spectacle_repository/ISpectacleRepository.java
package ro.mpp2024.concertpro.dao.repository.spectacle_repository;

import ro.mpp2024.concertpro.dao.model.Spectacle;
import ro.mpp2024.concertpro.dao.repository.IRepository;

import java.util.Date;

public interface ISpectacleRepository extends IRepository<Spectacle, Long> {
    Iterable<Spectacle> getAllByDate(Date date);
}
